espnet2/layers/stft.py

```diff
@@ -64,14 +66,23 @@ def forward(
         else:
             multi_channel = False
 
+        # NOTE(kamo):
+        #   The default behaviour of torch.stft is compatible with librosa.stft
+        #   about padding and scaling.
+        #   Note that it's different from scipy.signal.stft
+
         # output: (Batch, Freq, Frames, 2=real_imag)
         # or (Batch, Channel, Freq, Frames, 2=real_imag)
+        window = torch.hann_window(
+            self.win_length, dtype=input.dtype, device=input.device
+        )
```

Comment:
Is it OK not to optionize?

Reply:
We may use only hann, but I added just in case.